R/tbl-df.r

```diff
@@ -69,7 +69,7 @@ print.tbl_df <- function(x, ..., n = NULL, width = NULL, n_extra = NULL) {
   # Next, subset rows
   if (!missing(i)) {
     if (length(x) == 0) {
-      nr <- length(attr(x, "row.names")[i])
+      nr <- length(seq_len(nr)[i])
```

Comment:
Thanks. Can we keep the original code if `has_rownames()` is `TRUE` for the original data frame?